src/client/Client.java

```diff
@@ -106,6 +117,11 @@ private void updateFrame(int choiceNum) {
                     currScene = scene;
                 }
             }
+            if (oldScene.equals(currScene.name)) {
```

Comment:
Can you give some insight as to when this occurs?

Reply:
if the scene that was written in the code doesnt actually exist

Reply:
Ahh alright. Thanks